AmplifyPlugins/Analytics/AWSPinpointAnalyticsPlugin/Dependency/Pinpoint/Session/PinpointSession.swift

```diff
@@ -0,0 +1,101 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+class PinpointSession: Codable {
+    let sessionId: String
```

Comment:
nit: `SessionId` typealias?

Reply:
It can be done and I guess since it's an internal class it shouldn't be a problem, but I'm wondering: where do we draw the line when creating `typealias`es?

I mean this id is just a `String`, what's the reasoning behind favouring a typealias over the primitive?  🤔